modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/paths/RaylifeCreateAccount.path

```diff
@@ -15,6 +15,11 @@
 	<td>//div[contains(@class, 'quote-header') and contains(., '${title}')]/parent::div//button[contains(@id, 'purchase')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CREATE_NEW_ACCOUNT</td>
+	<td>//*[@class= 'align-items-center d-flex mr-3 mt-6 btn btn-secondary' and contains(.,'CREATE NEW ACCOUNT')]</td>
+	<td></td>
+</tr>
```

Comment:
Hi @itstali, it's not a good practice to use long attribute values (the class on this example), because that makes the path fragile and inconsistent. Anyway, I think that this path isn't necessary, since you are using a very general path, so maybe you can re-use the `Button#ANY` path. Could you check it, please?